set.mm

```diff
@@ -424241,18 +424241,21 @@ allows us to easily use the theorem in places where (in traditional
        Deductions have a label suffix of "d", especially if there are other
        forms of the same theorem (e.g., ~ pm2.43d ).  The labels for inferences
        usually have the suffix "i" (e.g., ~ pm2.43i ).  The labels of theorems
-       in "closed form" would have no special suffix (e.g., ~ pm2.43 ).  When
-       an inference is converted to a theorem by eliminating an "is a set"
+       in "closed form" would have no special suffix (e.g., ~ pm2.43 ) or, if
+       the standard label for the closed form is already in use, suffix "t"
+       (for "theorem" or "tautology", e.g., ~ ancomst or ~ nfimt ).  When an
+       inference is converted to a theorem by eliminating an "is a set"
        hypothesis, we sometimes suffix the closed form with "g" (for "more
```

Comment:
I think it's hard to understand for someone who hasn't seen an example of that. So maybe:
> When a theorem has a hypothesis of the form `A e. _V` and we prove a more general version where that hypothesis is replaced with an antecedent of the form `( A e. V -> ...`, then we suffix a "d" to the label of the original theorem, as in ~uniexg generalizing ~ uniex .